Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -0,0 +1,810 @@
+the documentation and build a full page tree.
+
+The Settings.cfg configuration file allows you to set theme variables, i.e. the
+project title, release version and the like.
+
+Of course, you can also use a README.rst instead of a README.md file to write
+the entire documentation in the same markup language.
+
+.. code-block:: none
+
+   .
+   ├── README.md
+   └── Documentation
+       ├── Includes.rst.txt
+       ├── Index.rst
+       ├── Settings.cfg
+       └── ..
+
+
+.. _readme-md:
+
+README.md
+---------
+
+Full documentation contains both a README.md and a Documentation/Index.rst file.
+To avoid redundancy in both places, the README.md in this case usually contains
+only a summary and links to all aspects of the project, i.e. the VCS
+repository, the published documentation and - if available - the TYPO3 Extension Repository (TER)
+page to guide the reader to the next steps. This could be for example:
+
+.. code-block:: md
+
+   # <project>
+
+   <abstract>
+
+   |                  | URL                                                           |
+   |------------------|---------------------------------------------------------------|
+   | **Repository:**  | https://<vcs-repository>                                      |
+   | **Read online:** | https://docs.typo3.org/p/<vendor>/<extension-key>/main/en-us/ |
+   | **TER:**         | https://extensions.typo3.org/extension/<extension-key>/       |
+
+or as README.rst alternatively:
+
+.. code-block:: rst
+
+   =========
+   <project>
+   =========
+
+   <abstract>
+
+   :Repository:  https://<vcs-repository>
+   :Read online: https://docs.typo3.org/p/<vendor>/<extension-key>/main/en-us/
+   :TER:         https://extensions.typo3.org/extension/<extension-key>/
+
+
+.. _readme-md-project:
+
+Project
+^^^^^^^
+
+The *project* placeholder contains the title of the project.
+
+Common values are in the official TYPO3 manuals
+
+#. `<Topic> Guide`, e.g. "Installation and Upgrade Guide",
+   for collections of articles on a specific topic
+#. `<Topic> Reference`, e.g. "TCA Reference",
+   for a complete encyclopedia
+#. `<Topic> Tutorial`, e.g. "Getting Started Tutorial",
+   for collections of tutorials on a specific topic
+
+in TYPO3 system extensions
+
+*  `TYPO3 CMS <Topic> (<package-name>)`, e.g. "TYPO3 CMS Extbase (typo3/cms-extbase)".
+
+and in TYPO3 third-party extensions
+
+*  `TYPO3 extension <extension-key>`, e.g. "TYPO3 extension \`mask\`".
+
+
+.. _readme-md-abstract:
+
+Abstract
+^^^^^^^^
+
+The *abstract* placeholder contains a short and precise description of the
+project with as many keywords as possible in as few sentences as possible. It
+helps the decision maker to quickly decide whether the project is worth
+considering and whether or not to read the full documentation. It should be
+aligned with the abstract of Index.rst and - if available - the description
+fields of ext_emconf.php and composer.json.
+
+
+.. _index-rst:
+
+Documentation/Index.rst
```

Comment:
I think this is not really the best content for a index.rst a much more modern aproach would be a tile structure like we have in TYPO3CMS-Guide-HowToDocument or TYPO3CMS-Tutorial-GettingStarted.

This strange outdated table like structures makes ppl afraid to use the start page for something creative. Beside the fact that we need a toctree (which can be hidden) we really dont need this.

Reply:
The meta data table contains duplicate information, but it has been greatly reduced, and there is also unique information such as the extension key/package name and authors. It also contains the unique abstract and a unique view of the expanded menu. Finally, all this information is presented in the main content area, so the reader has it at hand immediately. Especially on mobile devices, it makes a difference to put meta data somewhere in the frame of the page, e.g. at the very bottom of the footer, or at the very top of the content area.

As the academic style is used in most current documentations and i can also see use cases for it for coming documentations, especially in extension documentations, i want to establish a standard for it (this was my motivation, in fact). This does not keep you from (1) writing down a standard for Index.rst in this portal style and (2) not from applying it in all your documentations.